.travis.yml

```diff
@@ -14,6 +14,7 @@ matrix:
           env: SKELETON_VERSION="^4.0"
 
 before_install:
+    - export PACKAGES_ONLY_SF4="league/flysystem-bundle"
```

Comment:
We should revert the changes in .travis.yml

It is okey that the build partially fails.